sql/moz-fx-data-shared-prod/braze_external/users_previous_day_snapshot_v1/metadata.yaml

```
@@ -19,3 +19,5 @@ bigquery:
 scheduling:
   dag_name: bqetl_braze
   date_partition_parameter: null
+  parameters:
+  - "snapshot_interval:INT64:1"
```

ok but ... can you overwrite it if you hardcode it?

---

not saying you can't ... just not sure

---

you can't. you need to define available parameters in Airflow like this: https://github.com/mozilla/telemetry-airflow/blob/5254b94e51020991691a88fb09f8fe43565fc23d/dags/operational_monitoring_backfill.py#L32-L60
But currently the generation logic doesn't support this

---

actually one thing to try that could work:

```suggestion
  - "snapshot_interval:INT64:{{ dag_run.conf.get('snapshot_interval', 1) }}"
```